server/src/main/java/org/elasticsearch/common/util/Maps.java

```diff
@@ -284,18 +284,4 @@ static int capacity(int expectedSize) {
         return expectedSize < 2 ? expectedSize + 1 : (int) (expectedSize / 0.75 + 1.0);
     }
 
-    /**
```

Comment:
Not needed anymore with the way I do parsing. This method was added previously by me.